.eslintrc.js

```diff
@@ -47,5 +47,7 @@ module.exports = {
     ],
     '@typescript-eslint/no-var-requires': 0,
     'prettier/prettier': ['error'],
+    // used to fix error  Using exported name as identifier for default export
```

Comment:
This esline change actually shouldn't be needed after you change the name of the TS type `SingleShoppingList` to `ShoppingListProps`.

The error message is telling you that you're exporting two things with the same name, which is quite clearly a bad practice!